### ISSUE TITLE
issue with the ride of the train, wont drive

### ISSUE THREAD

**Issue by Ra78u6** [Xannosz/BetterMinecarts#3]
it keeps standing still, tryed with less carts, it wants to go forward but keeps struggling

this is the error 

[15:24:21] [Worker-Main-6/ERROR] [minecraft/TagLoader]: Couldn't load tag [betterminecarts:glowing_rail (from betterminecarts-1.20.1-1.8.1-47.1.0 (1).jar)] as it is missing following references: betterminecarts:glowing_rail (from betterminecarts-1.20.1-1.8.1-47.1.0 (1).jar)

this is the warn

[15:24:28] [Server thread/WARN] [minecraft/SynchedEntityData]: defineId called for: class net.minecraft.world.entity.vehicle.AbstractMinecart from class hu.xannosz.betterminecarts.utils.MinecartHelper
[15:24:28] [Server thread/WARN] [minecraft/SynchedEntityData]: defineId called for: class net.minecraft.world.entity.vehicle.AbstractMinecart from class hu.xannosz.betterminecarts.utils.MinecartHelper
[15:24:28] [Server thread/WARN] [minecraft/SynchedEntityData]: defineId called for: class net.minecraft.world.entity.vehicle.AbstractMinecart from class hu.xannosz.betterminecarts.utils.MinecartHelper
[15:24:28] [Server thread/WARN] [minecraft/SynchedEntityData]: defineId called for: class net.minecraft.world.entity.vehicle.AbstractMinecart from class hu.xannosz.betterminecarts.utils.MinecartHelper
[15:24:28] [Server thread/WARN] [minecraft/SynchedEntityData]: defineId called for: class net.minecraft.world.entity.vehicle.AbstractMinecart from class hu.xannosz.betterminecarts.utils.MinecartHelper

**Comment by Xannosz** [comment]
Hi, sorry for late response. The error is not a problem it just a missing but not used translation tag.

I try to reproduce the warn, but I can not. Some questions:
Can you try it without cars?
Can you send me a list from your other mods? (maybe it is an incompatibility)
Are you sure, your locomotive is the head of your train?

**Comment by Ra78u6** [comment]
yes srry for the log time, i didnt see it, it has the locomotive yes infront, and it looks if it has an issue with chunk borders or something
